models/Movie.js

```diff
@@ -11,46 +11,70 @@ Movie.init(
             primaryKey: true,
             autoIncrement: true,
         },
-        series_title: {
+        Poster_Link: {
             type: DataTypes.STRING,
             allowNull: false,
+            validate: {
+                isUrl: true,
+            },
         },
-        genre: {
+        Series_Title: {
             type: DataTypes.STRING,
             allowNull: false,
         },
-        released_year: {
+        Released_Year: {
             type: DataTypes.INTEGER,
             allowNull: false,
         },
-        overview: {
+        Certificate: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        Runtime: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        Genre: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        IMDB_Rating: {
+            type: DataTypes.NUMBER,
+            allowNull: false,
+        },
+        Overview: {
             type: DataTypes.TEXT,
```

Comment:
For consistency, this should be DataTypes.STRING

Reply:
Even tho .TEXT is an acceptable DataType, I think consistency across the code is important.